src/ripple/app/ledger/LedgerConsensus.h

```diff
@@ -56,16 +56,16 @@ class LedgerConsensus : public Traits
 
     virtual LgrID_t getLCL () = 0;
 
-    virtual void gotMap (TxSet_t const& map) = 0;
+    virtual void gotMap (Time_t const& now, TxSet_t const& map) = 0;
```

Comment:
Consensus actually deals with network times (`NetClock::time_point`) and internal times (`std::chrono::steady_clock'). The latter is used to measure the progress of consensus locally and independent of the network time.

@joelkatz, is it important for the internal time to be independent and have higher resolution?